cmd/operator-sdk/generate/csv.go

```diff
@@ -199,7 +199,7 @@ func writeCRDsToDir(crdPaths []string, toDir string) error {
 		}
 		typeMeta, err := k8sutil.GetTypeMetaFromBytes(b)
 		if err != nil {
-			return err
+			return fmt.Errorf("Error in %s : %v", p, err)
```

Comment:
```suggestion
			return fmt.Errorf("error in %s : %v", p, err)
```

Reply:
Thanks @camilamacedo86. I'll follow the same process in future to link the bugs.